TGS3.json

```diff
@@ -18,5 +18,6 @@
     ],
     "dlls": [
       "libmysql.dll"
```

Comment:
Invalid json.
Missing the , after "libmysql.dll

Reply:
Also, space vs tab indentation.
Make sure to use space for the new line you added.